domains/oslc-domains/pom.xml

```diff
@@ -20,12 +20,34 @@
     <packaging>jar</packaging>
     <name>Lyo :: Domains</name>
     <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
```

Comment:
All of these properties are already set by the parent. It will be hard to debug when we update a parent property and nothing changes because these properties override the parent ones.